compile-unitpacker-config-tools.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# GitHub Actions script to gather configuration from the local config.json file
+set -eux
+
+# Find tool configuration from config.json - Output: TOOL_CONFIG 
+TOOL_NAME="$(echo "$1" | sed 's/^\(.*\)-v.*$/\1/')"
+TOOL_CONFIG="$(jq '.tools | map(select(.name == "'"$TOOL_NAME"'"))[0]' config.json)"
+
+# Get the 'os' and 'tool' for unitpacker.
+CONFIG="$(jq -s '{os: .[0].os} + {tools: .[0].tools}' config.json "${TOOL_NAME}_config.json" | tr -d '\n')"
```

Comment:
I don't understand what `"${TOOL_NAME}_config.json"` does here. There's no such file (ex: multisig_config.json doesn't exist). `jq` doesn't write a file like that but even if it did, it wouldn't be used for anything.

```suggestion
CONFIG="$(jq -s '{os: .[0].os} + {tools: .[0].tools}' config.json | tr -d '\n')"
```

Reply:
You are right. I was testing something but this is not needed. It is update now.